test/cli/testprojinfo

```diff
@@ -310,6 +310,38 @@ echo 'Checks that ED50 to ETRS89 (12) is in the output (superseded transformatio
 $EXE -s EPSG:23030 -t EPSG:25830 --bbox -6,40,-5,41 --grid-check known_available --hide-ballpark --summary >>${OUT} 2>&1
 echo "" >>${OUT}
 
+echo 'Testing --list-crs | grep "EPSG:432\|EPSG:3263\|ESRI:103668\|OGC" | sort' >> ${OUT}
```

Comment:
Those tests are going to be fragile when the EPSG database is extended. Could you add more restrictive grepping to restrict to one or two max CRS in output ?

Reply:
No Problem

Reply:
done